requirements.txt

```diff
@@ -1,3 +1,3 @@
 ansible
 hpICsp
-hpOneView==5.0.0
+hpOneView==5.1.0
```

Comment:
safe to update when we have support for all the resources upgraded in the new Python SDK release. I think we should update this when we release the new library version.

Reply:
done